charts/internal/calico/templates/node/daemonset-calico-node.yaml

```diff
@@ -178,7 +178,7 @@ spec:
               cpu: 250m
               memory: 100Mi
             limits:
-              cpu: 800m
+              cpu: 4000m
```

Comment:
With gardener/gardener-extension-networking-calico#14 we raised the limit to 800m and the calico-node Pods which were cpu throttled at `500m` are now with cpu usage of 50m.
At least it looks to me like the high cpu usage of calico-node is a result of another issue.

Reply:
The hard part is to figure out the root cause, @zanetworker already helped a lot finding this issue. But is difficult.

We build the image, no prob